src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTestData.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Test.Cryptography;
+
+namespace System.Security.Cryptography.EcDsa.Tests
+{
+    // Note to contributors:
+    //   Keys contained in this file should be randomly generated for the purpose of inclusion here,
+    //   or obtained from some fixed set of test data. (Please) DO NOT use any key that has ever been
+    //   used for any real purpose.
+    //
+    // Note to readers:
+    //   The keys contained in this file should all be treated as compromised. That means that you
+    //   absolutely SHOULD NOT use these keys on anything that you actually want to be protected.
+    internal static class ECDsaTestData
+    {
+        // AllowExport|AllowPlainTextExport,  CngKeyCreationOptions.NOne, UIPolicy(CngUIProtectionLevels.None), CngKeyUsages.Decryption
+        public static byte[] Key_ECDiffieHellmanP256 =
```

Comment:
> there were shared (Cng + OpenSsl) tests added that use these

Where?  I just searched right now and only see hits in this file, and in files/directories named Cng.

45434b31 is the CNG dwMagic header for this blob type, it's very CNG specific, and not standard at all...